cmd/generate/config/rules/onfido.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+    "github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+    "github.com/zricethezav/gitleaks/v8/config"
+)
+
+func OnfidoAPIToken() *config.Rule {
+    // define rule
+    r := config.Rule{
+        RuleID:      "onfido-live-api-token",
+        Description: "Found an Onfido live API Token posing a risk of unauthorized access to identity verification services data, potentially leading to misuse of PII.",
+        Regex:       utils.GenerateUniqueTokenRegex(`(?:api_live(?:_[a-zA-Z]{2})?\.[a-zA-Z0-9-_]{11}\.[-_a-zA-Z0-9]{32})`, true),
+
+        Keywords: []string{
+            "onfido",
```

Comment:
I don't think this is necessary as the token will only match `api_live`.

Reply:
Done, removed the "onfido" keyword 👍